### ISSUE TITLE
Question about the project

### ISSUE THREAD

**Issue by yoyossef** [JustTemmie/steam-presence#1]
In the README it says:

> in addition to this, there's the Steam Deck, a handheld linux game "console".
>
> having discord constantly run in the background is a terrible idea considering how that's gonna lose you at least half an hour of battery life, in addition to the previous issues with linux.

Which is a valid point, but Discord still needs to run in the background if I understand it correctly, or at least when I launch the project on Deck it says that it needs to.

Am I missing something?

**Comment by JustTemmie** [comment]
yeah @akkornel is right, the readme file isn't exactly great, i'll fix it up later today - thanks for creating an issue
